<launch>
    <arg name="veh" default="starducks" doc="Name of vehicle. ex: megaman"/>

    <arg name="pkg_name" value="virtual_mirror_teddy"/>
    <arg name="node_name" default="virtual_mirror_node"/>

    <group ns="$(arg veh)">
        <remap from="virtual_mirror_node/image_in" to="camera_node/image/compressed" />
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" />
    </group>
</launch>
